transcription/sphinx.go

```diff
@@ -0,0 +1,57 @@
+// Package transcription implements functions for the manipulation and
+// transcription of audio files.
+package transcription
+
+import (
+	"bufio"
+	"encoding/json"
+	"os"
+	"os/exec"
+)
+
+//Transcription contains the transcription text and metadata of a transcription
+//job
+type Transcription struct {
```

Comment:
We probably want to use the same `TranscriptionResult` struct for IBM and Sphinx right? If so, I think we should further refactor so that `TranscriptionResult` is defined in `transcription.go`. We can add one or two non-util functions from `utils.go` to `transcription.go`, too.